app/shared/availability-view/TimelineGroups/TimelineGroup/AvailabilityTimeline/ReservationSlot.js

```diff
@@ -1,7 +1,26 @@
-import React from 'react';
+import queryString from 'query-string';
+import React, { PropTypes } from 'react';
 
 import utils from '../utils';
 
-export default function ReservationSlot() {
-  return <div className="reservation-slot" style={{ width: utils.getTimeSlotWidth() }} />;
+ReservationSlot.propTypes = {
+  begin: PropTypes.shape({
+    format: PropTypes.func.isRequired,
+  }).isRequired,
+  resourceId: PropTypes.string.isRequired,
+};
+export default function ReservationSlot(props) {
+  const query = queryString.stringify({
+    begin: props.begin.format(),
+    reserve: true,
+  });
+  return (
+    <a
```

Comment:
- [ ] This should be a react-router `Link` so the whole page isn't refreshed when the link is clicked.